### DISCUSSION TITLE
New User Onboarding Template

### DISCUSSION THREAD

**Discussion by ContinuousJay** [KelvinTegelaar/CIPP#2158] [Ideas]
It would be great to have the ability to create templates on a per tenant basis or global for new user onboarding so we can include by default additional attributes such as officeLocation and require them to be mandatory or not.

**Comment by BNWEIN** [comment]
I think the best way to do this would be to add some default options under the "User Settings" Section. Here you could add the default attributes you want available during your "New User" or "Edit User" process.  If you are a sponsor, could be worth raising an FR! :)

**Reply by ContinuousJay** [reply]
There is no such option in user settings.

**Reply by BNWEIN** [reply]
Yes, i was suggesting the best place for the change to go if the FR gets picked up :)
